tracer/test/Datadog.Trace.Tests/Telemetry/TelemetrySettingsTests.cs

```diff
@@ -322,6 +322,19 @@ public void V2Enabled_DisabledByDefaultInAas(string value, bool expected)
             settings.V2Enabled.Should().Be(expected);
         }
 
+        [Theory]
+        [InlineData("0", false)]
+        [InlineData(null, false)]
+        [InlineData("", false)]
+        [InlineData("1", true)]
+        public void V2Enabled_DisabledByDefaultInCIVisibility(string value, bool expected)
```

Comment:
If only it _were_ disabled by default 😛 based on your code in `TelemetrySettings`, I would expect this to be _enabled_ by default?

Reply:
I've reverted the telemetry settings changes in https://github.com/DataDog/dd-trace-dotnet/pull/4596/commits/172031b6787cc51aabf963645844617e8e78d65b